de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537820_sys_gh3427_attribute_window.sql

```diff
@@ -0,0 +1,35 @@
+-- 2019-12-04T10:11:33.327Z
```

Comment:
this whole script is pointless and basically does nothing related with it's name.
* it changes some preferences => don't know what prefs. are those but i don't want them to be changed on each system we have
* it changes the language for user SuperUser => don't want that to be changed to all systems
* changes some attributes => don't want that to be changed to all systems